angular-workspace/projects/ni/nimble-angular/src/directives/breadcrumb-item/nimble-breadcrumb-item-router-link-with-href.directive.ts

```diff
@@ -1,5 +1,7 @@
-import { Directive, Input } from '@angular/core';
-import { RouterLinkWithHref } from '@angular/router';
+import { LocationStrategy } from '@angular/common';
+import { Directive, ElementRef, HostListener, Injector, Input } from '@angular/core';
+import { ActivatedRoute, Router, RouterLinkWithHref } from '@angular/router';
```

Comment:
Looks like RouterLink is a standalone directive in angular 14: https://github.com/angular/angular/blob/14.2.x/packages/router/src/directives/router_link.ts#L308

Very open ended but can we use the host directives feature to compose Angular's directive instead of extending their directive? https://angular.io/guide/directive-composition-api#directive-composition-api

It's possible this bug needs to be addressed by extending Angular's directive and maybe the host directives feature could be used for our other router link usages instead.

Reply:
I'd have to do some more investigation before I could say with much confidence which directives would make sense to use host directives and what that would simplify within nimble. However, host directives are an Angular 15 feature. Since we're on Angular 14, there isn't anything for us to do at this point in time.

Should I file a tech debt issue so that we'll remember to evaluate host directives once we've upgraded to Angular 15?

Reply:
> Should I file a tech debt issue so that we'll remember to evaluate host directives once we've upgraded to Angular 15?

Gosh darnit, good catch. Yea that sounds like a good idea

Reply:
Created https://github.com/ni/nimble/issues/1032